packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx

```diff
@@ -90,3 +91,49 @@ test(integrationTest('Service Multi Execution Editor'), async () => {
   await waitFor(() => getByText(changeDialog, 'QA'));
   await waitFor(() => getByText(changeDialog, 'Change'));
 });
+
+const pluginManager = LegendStudioPluginManager.create();
+pluginManager.usePresets([new QueryBuilder_GraphManagerPreset()]).install();
+
+test(
+  integrationTest('Test Enternal Format Service Test Parameter Setup'),
+  async () => {
+    const MOCK__editorStore = TEST__provideMockedEditorStore({ pluginManager });
+    const renderResult = await TEST__setUpEditorWithDefaultSDLCData(
+      MOCK__editorStore,
+      {
+        entities: TEST_DATA__ExternalFormatServiceEntities,
+      },
+    );
+
+    MockedMonacoEditorInstance.getValue.mockReturnValue('');
+    await TEST__openElementFromExplorerTree(
+      'demo::externalFormat::flatdata::simple::service::FlatdataWithM2MChainingMerged',
+      renderResult,
+    );
+    const editorGroup = await waitFor(() =>
+      renderResult.getByTestId(LEGEND_STUDIO_TEST_ID.EDITOR_GROUP),
+    );
+    await waitFor(() => getByText(editorGroup, 'Test'));
+    fireEvent.click(getByText(editorGroup, 'Test'));
+    const serviceTestEditor = await waitFor(() =>
+      renderResult.getByTestId(LEGEND_STUDIO_TEST_ID.SERVICE_TEST_EDITOR),
+    );
+    await waitFor(() => getByText(serviceTestEditor, 'Setup'));
+    fireEvent.click(getByText(serviceTestEditor, 'Setup'));
+    const serviceTestSetupEditor = await waitFor(() =>
+      renderResult.getByTestId(
+        LEGEND_STUDIO_TEST_ID.SERVICE_TEST_EDITOR__SETUP__PARAMETERS,
+      ),
+    );
+    expect(
+      MOCK__editorStore.tabManagerState
+        .getCurrentEditorState(ServiceEditorState)
+        .testableState.selectedSuiteState?.testStates[0]?.setupState.getBindingWithParamFromQuery(),
+    ).toHaveLength(2);
```

Comment:
can we do more checks than just the number of pairs. check the binding and  param expected values ?